codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/protocols/serialize/XmlBindingTraitSerializerGenerator.kt

```diff
@@ -469,6 +472,9 @@ class XmlBindingTraitSerializerGenerator(
     private fun OperationShape.operationXmlMembers(): XmlMemberIndex =
         XmlMemberIndex.fromMembers(httpBindingResolver.requestMembers(this, HttpLocation.DOCUMENT))
 
+    private fun OperationShape.serverOutputXmlMembers(): XmlMemberIndex =
```

Comment:
what do you think about renaming these to `requestBodyMembers` and `responseBodyMembers`?